config/components/manager-ocp/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
+- manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue
```

Comment:
do we actually use this image name?

Reply:
https://github.com/openshift/kubernetes-sigs-kueue/pull/46/commits/21dca737986f6295f0acae1e262f0c29d2bd2052#diff-7f74dfdda77120ae6320a38cdae2a87d4ce88a703574b15f13822f9d47abb49dR30

In e2e tests, we end up replacing it with `IMAGE_TAG`.